cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/BuildCommand.java

```diff
@@ -279,7 +279,7 @@ public void execute() {
                 // clean the target directory(projects only)
                 .addTask(new CleanTargetDirTask(isPackageModified, buildOptions.enableCache()), isSingleFileBuild)
                 // Run build tools
-                .addTask(new RunBallerinaPreBuildToolsTask(outStream))
+                .addTask(new RunBuildToolsTask(outStream))
```

Comment:
Shall we rename it as `RunPreBuildToolsTask` as there might be post build task as well in future(As I remember they were mentioning something in  earlier discussion)?  IMO, `RunPreBuildToolsTask` task makes sense than `BuildToolsTask` as it is run before the build

Reply:
I removed the prefix `pre` since we haven't used it anywhere in the design, discussions, or, in the code other than here, in the task name. Anyway, let's discuss and come up with a name in the code review.